### PR TITLE
Added some NSE modules

### PR DESCRIPTION
**Pull request by arsenius7**
* nix/nse/brainbuilder
 * nix/nse/voxcell

### CODE REVIEW

File: bbp/modules/default.nix [hunk 1]
```diff
@@ -2392,7 +2414,7 @@ let
                 morphscale
                 morphsyn
                 muk
-                placement-algorithm
+                pybinreports
```
Comment (arsenius7) [inline bbp/modules/default.nix]: We were planning to bury 'pybinreports' soon (to use Brion exclusively for binary report access).
Please let me know why it needs to be distributed, especially as a module.
Thanks.

Comment (tristan0x) [reply bbp/modules/default.nix]: `git blame` tells me that @adevress added this in 3f2d8759c81a38dff47da2375e1f5877a5fabb50

Comment (adevress) [reply bbp/modules/default.nix]: Hi Arseny,

pybinreports has been added rather recently for the need of Christian and its simplification workflow (with bglibpy) . I have nothing against removing it if you can guarantee me that Christian scripts do not need it anymore :)

Adrien

Comment (arsenius7) [reply bbp/modules/default.nix]: OK, I'll check it with Christian then, just wanted to know where this one comes from.
You mean Christian O'Reilly, right?

**Comment by adevress** [comment]
Christian Rossert

On February 1, 2018 11:32:55 AM GMT+01:00, "Arseny V. Povolotsky" <notifications@github.com> wrote:
>arsenius7 commented on this pull request.
>
>
>
>> @@ -2392,7 +2414,7 @@ let
>                 morphscale
>                 morphsyn
>                 muk
>-                placement-algorithm
>+                pybinreports
>
>OK, I'll check it with Christian then, just wanted to know where this
>one comes from.
>You mean Christian O'Reilly, right?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/BlueBrain/bbp-nixpkgs/pull/624#discussion_r165313986

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request merged**